README.md

```diff
@@ -186,6 +186,9 @@ After about 30-60 seconds the crons should have completed and you should have da
  If a snapshot fails due to change in the UI, try updating the snapshot from the failed test from cypress controlled 
  browser 'compare snapshot' and a pop up appears to `Update Snapshot` 
 
+## Jest Testing
+`docker exec -it webpack_watcher npm test` will run the test suite for the front-end React application. [Jest](https://jestjs.io/) is the testing framework used. More tests (including back-end tests) are planned for the future.
```

Comment:
Some of this info should probably be moved under "Testing" but before "Cypress Testing" as a kind of introduction. I'm thinking it could say something like this:

"The application currently uses two frameworks for front-end testing: Cypress and Jest. There are plans to implement back-end tests in the future."

And then the "Jest Testing" section could say something like this:

"Other front-end tests leverage the [Jest framework]((https://jestjs.io/)). To run the Jest test suite, execute the command `docker exec -it webpack_watcher npm test`."

Any thoughts about that? Currently the language just doesn't fit together well.